tests/wpt/mozilla/tests/css/multiple_backgrounds.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="match" href="multiple_backgrounds_ref.html">
+    <style type="text/css">
+        /* Example from
+           https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Background_and_Borders/Using_CSS_multiple_backgrounds
+           (Public domain)
+
+           Rust logo from https://www.rust-lang.org/logos/rust-logo-256x256.png,
+           licensed as CC-BY-SA, owned by Mozilla
+        */
+        #multibg {
+            width: 700px;
+            height: 400px;
+            background: url(rust-logo-256x256.png) no-repeat bottom right / 256px 256px,
+                        url(bubbles.png) no-repeat left / 700px 100%,
```

Comment:
I'm not actually sure about the license for bubbles.png. MDN has CC-BY-SA for the text and public domain for the code. The image is used by the code, but I'm not sure what it counts as. We can attribute the logo to the MDN developers to satisfy the CC-BY-SA license, but I'm not sure if that makes the entire testsuite/codebase a derivative work. Probably not? cc @larsbergstrom